### PR TITLE
[4.0][Cassiopeia][CS] Line-breaks, indents and spaces

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/26759 .

### Summary of Changes
Consistent line-breaks, indents, and spacing across all Cassiopeia template SCSS using the following...

```JSON
"end_with_newline": true,
"indent_char": " ",
"indent_size": 2,
"newline_between_rules": true,
"selector_separator_newline": true,
"space_around_combinator": true,
```

### Testing Instructions
Code review

### CODE REVIEW

File: templates/cassiopeia/scss/blocks/_layout.scss [hunk 1]
```diff
@@ -90,7 +91,8 @@ header {
 }
 
 .container-sidebar-left,
-.container-sidebar-right, {
+.container-sidebar-right,
+  {
```
Comment (Quy) [inline templates/cassiopeia/scss/blocks/_layout.scss]: Revert and remove comma.

Comment (ciar4n) [reply templates/cassiopeia/scss/blocks/_layout.scss]: Thank you @Quy. Well spotted.

**Comment by Quiviro** [comment]
I have tested this item :white_check_mark: successfully on 539a64c279ac9fa79a848475f998fd6cdf8b6af8<br /><br />Code reviewed
#PBF19es<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26794">issues.joomla.org/tracker/joomla-cms/26794</a>.</sub>

**Comment by Quiviro** [comment]
I have tested this item :white_check_mark: successfully on 539a64c279ac9fa79a848475f998fd6cdf8b6af8<br /><br />Code reviewed
#PBF19es<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26794">issues.joomla.org/tracker/joomla-cms/26794</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 539a64c279ac9fa79a848475f998fd6cdf8b6af8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26794">issues.joomla.org/tracker/joomla-cms/26794</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26794">issues.joomla.org/tracker/joomla-cms/26794</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**